### DISCUSSION TITLE
language file

### DISCUSSION THREAD

**Discussion by shaycqs** [RedShyGuy/Vapecord-ACNL-Plugin#97] [Q&A]
when i start the game it tells me that there’s an error because i don’t have any language files. How can i solve this problem? Or where can i find language files?

**Comment by RedShyGuy** [comment]
You need to install Vapecord correctly like the Installation Instructions tell you. Move all contents of the zip to your sdcard and replace files when prompted

**Reply by shaycqs** [reply]
okay i’ll try again! thanks
